### ISSUE TITLE
Comment tags in HTML are not honoured.

### ISSUE THREAD

**Issue by willhartman** [rse/node-inline-assets#6]
If a script tag is commented out then the file backing it might not exist.

The utility still tries to grab the file even though it isn't needed for the HTML to render. 

This causes an error which can be resolved by deleting the line instead of commenting it out.

The utility should ignore script tags that are inside HTML comment tags.

**Comment by rse** [comment]
Yes, I agree. But without a real parser this cannot be achieved easily. For a later version I'll check whether we can use an HTML Parser which generates an AST and work on this instead.
